apps/cowswap-frontend/src/modules/fiatAmount/apis/getCowProtocolFiatPrice.ts

```diff
@@ -0,0 +1,32 @@
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { Token } from '@uniswap/sdk-core'
+
+import { USDC } from 'legacy/constants/tokens'
+
+import { getCowProtocolNativePrice } from './getCowProtocolNativePrice'
+
+export async function getCowProtocolFiatPrice(
```

Comment:
Shouldnt it say `Usd` instead of Fiat? It sets the expectations to the user of what is that `Promise<number>` (is definetly not EUR)

Reply:
Makes sense, renamed